lib/rucio/daemons/storage/consistency/actions.py

```diff
@@ -0,0 +1,805 @@
+    reftime = datetime.strptime(reftimestamp, '%Y_%m_%d_%H_%M')
+    runs = {}
+    for path in files:
+        fn = path.rsplit("/", 1)[-1]
+        if os.stat(path).st_size > 0:
+            r, timestamp, typ, ext = parse_filename(fn)
+            filetime = datetime.strptime(timestamp, '%Y_%m_%d_%H_%M')
+            fileagedays = (reftime - filetime).days
+            if r == rse:
+                # if the RSE was X, then rses like X_Y will appear in this list too,
+                # so double check that we get the right RSE
+                runs.update({path: fileagedays})
+
+    return {k:v for k, v in sorted(runs.items(), reverse=True)}
+
+def list_runs(path, rse, nlast=0):
+    files = glob.glob(f"{path}/{rse}_*_stats.json")
+    runs = []
+    for path in files:
+        fn = path.rsplit("/", 1)[-1]
+        if os.stat(path).st_size > 0:
+            r, timestamp, typ, ext = parse_filename(fn)
+            if r == rse:
+                # if the RSE was X, then rses like X_Y will appear in this list too,
+                # so double check that we get the right RSE
+                runs.append(path)
+    if nlast == 0:
+        nlast = len(runs)
+    return sorted(runs, reverse=False)[-nlast:]
+
+def list_unprocessed_runs(path, rse, nlast=0):
+    files = glob.glob(f"{path}/{rse}_*_stats.json")
+    unproc_runs = []
+    for path in files:
+        fn = path.rsplit("/", 1)[-1]
+        if os.stat(path).st_size > 0:
+            r, timestamp, typ, ext = parse_filename(fn)
+            if r == rse:
+                # if the RSE was X, then rses like X_Y will appear in this list too,
+                # so double check that we get the right RSE
+                if not was_cc_attempted(path):
+                    unproc_runs.append(timestamp)
+    if nlast == 0:
+        nlast = len(unproc_runs)
+    return sorted(unproc_runs, reverse=True)[-nlast:]
+
+def was_cc_attempted(stats_file):
+    try:
+        f = open(stats_file, "r")
+    except:
+        print("get_data: error ", stats_file)
+        return None
+    stats = json.loads(f.read())
+    cc_dark_status = ''
+    cc_miss_status = ''
+    if "cc_dark" in stats or "cc_miss" in stats:
+        return True
+    else:
+        return False
+
+def was_cc_processed(stats_file):
+    try:
+        f = open(stats_file, "r")
+    except:
+        print("get_data: error ", stats_file)
+        return None
+    stats = json.loads(f.read())
+    cc_dark_status = ''
+    cc_miss_status = ''
+    if "cc_dark" in stats:
+        if "status" in stats['cc_dark']:
+            cc_dark_status = stats['cc_dark']['status']
+    if "cc_miss" in stats:
+        if "status" in stats['cc_miss']:
+            cc_miss_status = stats['cc_miss']['status']
+    if cc_dark_status == 'done' or cc_miss_status == 'done':
+        return True
+    else:
+        return False
+
+
+def process_dark_files(path, rse, latest_run, maxdarkfraction, max_files_at_site, oldenough_run):
```

Comment:
Inconsistent use of `_`.

Reply:
I don't understand this comment. Can you elaborate, please?

Reply:
You have the variable  `max_files_at_site`, which delimits the words with a `_`. At the same time, you have the variable `maxdarkfraction`, without the word delimiter. I would suggest to use the word delimiter consistently, e.g.:
- `max_dark_fraction`
- `old_enough_run`

Reply:
Done.